src/main/java/ie/cit/teambravo/cardsec/CardsecApplication.java

```diff
@@ -2,13 +2,16 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.cache.annotation.EnableCaching;
 
+@EnableCaching
 @SpringBootApplication(scanBasePackageClasses = {
-        CardsecApplication.class
+		CardsecApplication.class
```

Comment:
not sure what's with your formatter but I don't like it. Too much whitespace.

Reply:
Ya can't figure out why... Using the ecliose formatter. It looks fine in intellij